### DISCUSSION TITLE
PRINTING: How can I remove the white border?

### DISCUSSION THREAD

**Discussion by niccolov** [photobooth-app/photobooth-app#213] [General]
I was able to install  and make it working with Raspberry bookworm on a Raspabberry 4.
I've configured a Pi Camera 3 and a Canon Selphy CP910.
Everything works fine.... but I've a little problem. The photo I'm printing has big white borders on the side.
Smaller on top and bottom, bigger on left and right.

I used the following command to print:
lp -d Canon_CP910_ipp -o landscape -o fit-to-page {filename}

How can I remove these borders?
And where I should put the frames to use for the image to print?

**Comment by mgineer85** [comment]
Please try to add the media option to your command:
```
lp -d Canon_CP910_ipp -o media=Postcard.fullbleed -o landscape -o fit-to-page {filename}
```

There are other commands that could be helpful - not sure if they work as is, but give you context for further research.
```
lpoptions -l Canon_CP910_ipp 
lpoptions -p Canon_CP910_ipp -o StpiShrinkOutput=Expand
lpoptions -p Canon_CP910_ipp -o StpBorderless=True
```

In general printing is very specific to the printer/system so the support is limited because the hardware is also not available to me. Please try to google on this topic if it is not solved. When you have solved, add the information here would be awesome.



Regarding the use of images in frames I improved the documentation:
https://photobooth-app.org/setup/mediaprocessing/#use-custom-files-for-postprocessing
and
https://photobooth-app.org/setup/mediaprocessing/#frame-image

**Reply by niccolov** [reply]
WOW the following command worked fine:
lp -d Canon_CP910_ipp -o media=Postcard.fullbleed -o landscape -o fit-to-page {filename}

The only problem now is that is a little bit too much... I'm loosing something on the right and left border. 
I've to work on it. Do you have any idea? 

But generally I'm very happy that it's working fine.
Regarding the frames, thanks for the improvement. Now it's more clear and I will try something.

Ciao

**Reply by mgineer85** [reply]
To have it improved ensure the aspect ratio of images and printer match.
But even if the aspect ratio of printer and images match perfectly you always want to have the image printed little larger to ensure there is no white area any more even in case the paper is drawn a little tilted in the printer from the tray due to mechanical tolerances...
